src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingModifiedFileEntry.ts

```diff
@@ -92,8 +91,6 @@ export abstract class AbstractChatEditingModifiedFileEntry extends Disposable im
 	) {
 		super();
 
-		this.originalURI = ChatEditingTextModelContentProvider.getFileURI(_telemetryInfo.sessionId, this.entryId, modifiedURI.path);
```

Comment:
Notebook will not have a text counter part, and the URI will be different as well.